src/cdrizzleapi.c

```diff
@@ -228,47 +536,38 @@ tdriz(PyObject *obj UNUSED_PARAM, PyObject *args, PyObject *keywords) {
     p.exposure_time = expin;
     p.weight_scale = wtscl;
     p.fill_value = fill_value;
+    p.fill_value2 = fill_value2;
```

Comment:
One letter variable names should be avoided for things other than indices.  I recommend changing `p` to a more descriptive variable name.